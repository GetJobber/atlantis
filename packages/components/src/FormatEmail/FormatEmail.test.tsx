import React from "react";
import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react";
import { FormatEmail } from ".";

afterEach(cleanup);

it("renders a FormatEmail", () => {
  const tree = renderer
    .create(<FormatEmail email="email@address.me" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders a FormatEmail in an address tag", () => {
  const tree = renderer
    .create(<FormatEmail email="email@address.me" />)
    .toJSON();
  expect(tree.type).toBe("address");
});
