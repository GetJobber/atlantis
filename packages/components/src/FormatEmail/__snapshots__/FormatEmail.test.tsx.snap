// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a FormatEmail 1`] = `
<address
  className="email"
>
  email@address.me
</address>
`;
